DOCUMENTATION.md

```diff
@@ -0,0 +1,127 @@
+## Table of Contents
+* [Description](#description)
+* [MQTTClient](#mqttclient)
+* [connect](#connect)
+* [close](#close)
+* [startForever](#startforever)
+* [start](#start)
+* [stop](#stop)
+* [setUplinkCallback](#setuplinkcallback)
+  * [uplinkCallback](#uplinkcallback)
+* [setConnectCallback](#setconnectcallback)
+  * [connectCallback](#connectcallback)
+* [setDownlinkCallback](#setpublishcallback)
+  * [downlinkCallback](#publishcallback)
+* [setCloseCallback](#setclosecallback)
+  * [closeCallback](#closecallback)
+* [send](#send)
+* [License](#license)
+
+## Description
+
+This package provides you an easy way to connect to The Things Network via MQTT. Take note that, you'll first need to create an application with a device to run the constructor of the MQTT client because you need to provide, an applicationID and a deviceID.
+First include the package in your file like this:
+```python
+from ttnmqtt import MQTTClient as mqtt
+```
+
+### MQTTClient
+
+The class constructor can be called following this scheme:
+```python
+mqtt(appID, appAccessKey, [mqttAddress])
+```
+- `appID`: **string**  this the name you gave your application when you created it.
+- `appAccessKey`: **string**  it can be found at the bottom of your application page under **ACCESS KEYS**.
+- `mqttAddress`: **string**  this the address of the handler to which your application was registered. If you registered to a personnal or private handler, please provide the address, if not you don't need to provide this argument. Please make sure you also give the port number when providing the mqttAddress.
+All the above informations can be found in your The Things Network console.
+The constructor returns an **MQTTClient object** set up with your application informations, ready for connection.
+
+### connect
+This function connect your client, in case you closed it and which to open the connection once again.
+```python
+client.connect()
+```
+
+### close
+Disconnects the MQTT client from which we call the method. Also able to stop a forever loop in case the client was running on a loop launched by the `startForever()` method.
+```python
+client.close()
+```
+
+### startForever
```

Comment:
I don't think we need both `start()` and `startForever()`. Let's remove one of them, and let the developer choose which one to build on top of